.circleci/config.yml

```diff
@@ -47,6 +71,22 @@ workflows:
           filters:
             branches:
               only: "2.0"
+  aws_destroy_daily:
+    triggers:
+      - schedule:
+          # run every day at 10pm -- note: use spaces, not tabs
+          cron: "0 22 * * *"
+          filters:
+            branches:
+              only:
+                - "1.8"
```

Comment:
Can we make this `only: "2.0"` to match the existing 2.0-only jobs? I think Circle will figure it out if the 1.8 branch also has a config with `only: "1.8"` in it